src/components/DetailsPage.jsx

```diff
@@ -0,0 +1,40 @@
+import '../styling/DetailsPage.css'
+// eslint-disable-next-line
```

Comment:
- [ ] [OPTIONAL]  I noticed that you disabled **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1:

Reply:
Thank you so much for the feedback! I will, and also thanks for letting me know that the proper way is not to merge feature branches until I get the general review, I do appreciate it, have a nice day!

Reply:
Done!